### PR TITLE
fix sync block error

### PR DESCRIPTION
**Pull request by jerry-yu**
fix sync block error when in specific height 21

### CODE REVIEW

File: cita-executor/src/postman.rs [hunk 1]
```diff
@@ -305,7 +305,7 @@ impl Postman {
                 for proto_block in sync_res.take_blocks().into_iter() {
                     let open_block = OpenBlock::from(proto_block);
                     if !self.backlogs.insert_synchronized(open_block) {
-                        return false;
+                        continue;
```
Comment (keroro520) [inline cita-executor/src/postman.rs]: Could you add some more description for this PR?
What the problem is if `return false` here.

And can we add a test to reproduce this problem?

Comment (jerry-yu) [reply cita-executor/src/postman.rs]: you can try when node0's height is 21, node1 sync block from node0.
when node1's height reach 20, network always feed executor [20,21] block,but inserting block 20 return false,so block 21 can't be executed

Comment (keroro520) [reply cita-executor/src/postman.rs]: > when node1's height reach 20, network always feed executor [20,21] block

Why `network` doesn't feed `executor` `[21]` when retry?

I am curious what's preventing adding a test to reproduce this problem?

Comment (keroro520) [reply cita-executor/src/postman.rs]: Another question: why `21` is special?

Comment (jerry-yu) [reply cita-executor/src/postman.rs]: that is another question. here,why not deal with next block, just because of prev block duplicated

**Comment by keroro520** [comment]
I will dismiss my RequestChange.
But I cannot ensure that the so-called 21-problem is resulted from "return early for duplicated". So I will remove myself from reviewers.

**Comment by jerry-yu** [comment]
from whatever view, directly returning false is risky  and not friendly for optimize

**Pull request merged**